vunit/sim_if/modelsim.py

```diff
@@ -410,9 +410,9 @@ def encode_generic_value(value):
     """
     s_value = str(value)
     if " " in s_value:
-        return '{"%s"}' % s_value
+        return f'{"{s_value!s}"}'
```

Comment:
Should be `return f'{{"{s_value!s}"}}'`

Reply:
Thanks! I applied the fix and force-pushed.